### PR TITLE
Test with ruby 3

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: .github/workflows/ruby_ci.yml [hunk 1]
```diff
@@ -1,84 +1,22 @@
...
-        with:
-          name: logs
-          path: log/*.log
-          retention-days: 5
+    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@v0
+    with:
+      plugin: foreman_remote_execution
+      # https://github.com/theforeman/foreman/pull/9871
+      foreman_repository: ofedoren/foreman
+      foreman_version: feat-36849-add-ruby-3-ga
```
Comment (ofedoren) [inline .github/workflows/ruby_ci.yml:22]: It should probably now test against https://github.com/theforeman/foreman/pull/9989

**Comment by adamruzicka** [comment]
The failing test should be fixed by https://github.com/theforeman/foreman-tasks/pull/738

**Comment by ekohl** [comment]
@ofedoren can you rebase?

**Comment by adamruzicka** [comment]
Rebased and switched to https://github.com/theforeman/foreman/pull/9989

**Comment by ofedoren** [comment]
I guess it can be closed. We run Foreman tests on Ruby 3, so it's auto-included anywhere the shared workflow being used.

**Pull request closed without merging**